README.md

```diff
@@ -1 +1,45 @@
-# Hello-microv
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a demo project for microverse.
+
+
+## Built With
+
+- HTML 5
+- CSS 3
+
+
+## Getting Started
+
+**Just clone the project by runing the `git clone git@github.com:abreham1222/Hello-Microverse.git` command in your command line, and you are done.**
+
+
+## Authors
+
+👤 **Abreham Awoke**
+
+- GitHub: [@githubhandle](https://github.com/abreham1222)
+- Twitter: [@twitterhandle](https://twitter.com/Abreham1222)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/abreham1222)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] your readme file looks professional 💯 , kindly fix the license link to make it perfect